### PR TITLE
Fix spacer deselection after dragging

### PR DESCRIPTION
**Pull request by talldan**
## Description
Fixes a small issue with the spacer in `master` that seems to be a regression.

Fix required deleting some code, so seems like a win!

## How has this been tested?
Adds an e2e test so this never happens again 😈.

Manual steps to repro:
1. Add a spacer block. 
2. Click and drag the drag handle to resize the spacer block

Expected behaviour (what happens on this branch)
When releasing after resizing using the drag handle, the spacer block should remain selected.

Actual behaviour (what happens in `master`)
When releasing after resizing using the drag handle, the spacer block becomes deselected. The grey area denoting the size of the spacer becomes invisible, which can be confusing if the spacer is the last block.

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/e2e-test-utils/README.md [hunk 1]
```diff
@@ -141,6 +141,21 @@ _Parameters_
...
 Disables Pre-publish checks.
 
+<a name="dragAndResize" href="#dragAndResize">#</a> **dragAndResize**
+
+Clicks an element, drags a particular distance and releases the mouse button.
+
+_Parameters_
+
+-   _element_ `Object`: The puppeteer element handle.
+-   _delta_ `number`: Object containing movement distances.
```
Comment (noisysocks) [inline packages/e2e-test-utils/README.md]: Should this say `Object`?

```suggestion
-   _delta_ `Object`: Object containing movement distances.
```

File: packages/e2e-test-utils/src/drag-and-resize.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Clicks an element, drags a particular distance and releases the mouse button.
+ *
+ * @param {Object} element The puppeteer element handle.
+ * @param {number} delta   Object containing movement distances.
```
Comment (noisysocks) [inline packages/e2e-test-utils/src/drag-and-resize.js]: Should this say `Object`?

```suggestion
 * @param {Object} delta   Object containing movement distances.
```

(http://usejsdoc.org is down for me right now and I can't remember if, for this type of argument, you're supposed to say `{Object}` or leave off the type altogether.)

**Review by noisysocks — APPROVED** [review]
Tests well! Thanks for adding tests ✌️😀

File: packages/e2e-test-utils/src/drag-and-resize.js (hunk 2 above)
Comment (talldan) [reply packages/e2e-test-utils/src/drag-and-resize.js]: Yep, it was an oversight. Refactored the parameters a few times, and at some point introduced an error. I've committed the suggestion, thanks for spotting 😄

**Pull request merged**